misc/packaging/acrn-hypervisor.postinst

```diff
@@ -125,4 +125,7 @@ sed -i '$a GRUB_TIMEOUT=20' ${filename}
 sync
 update-grub
 
+# Enable "systemd-networkd" service
+systemctl enable systemd-networkd
```

Comment:
Is there a risk of conflict with `NetworkManager` here (which I assume is enabled by default on Debian and Ubuntu? If so, should we also disable it as part of this postinst script?

@tttech-industrial-buchsbaum , thoughts?

Reply:
@gvancuts Thanks for pointing out this potential risk.
Previously, we didn't run "systemctl disable NetworkManager" to disable NetworkManager and didn't observe issues with it.

@NanlinXie  Would you pls help let the virtio-net feature owner to confirm whether it is required to disable NetworkManager? If yes, I could help update it in this PR. Thanks!

Reply:
@gvancuts, from our testing result, we didn't explicitly enable/disable NetworkManger, it works in both conditions.

Reply:
After some discussion, let's decouple ACRN package and network staff, thus will not add network configuration into ACRN deb.  We could document systemd-networkd steps for now, and in future, let's add one optional deb package for network manager, then user could choose to install or customize based on his/her own solution.